scripts/pylib/twister/twisterlib/runner.py

```diff
@@ -604,18 +689,19 @@ def process(self, pipeline, done, message, lock, results):
             logger.debug("build test: %s" % self.instance.name)
             res = self.build()
             if not res:
-                self.instance.status = "error"
+                self.instance.status = Status.ERROR
                 self.instance.reason = "Build Failure"
                 pipeline.put({"op": "report", "test": self.instance})
             else:
                 # Count skipped cases during build, for example
                 # due to ram/rom overflow.
-                if  self.instance.status == "skipped":
+                # FIXME: ram/rom are now errors!
```

Comment:
If you cannot fix this in this PR, please write at least what is expected and what alternatively could be changed in the future.